code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -133,18 +134,19 @@
 	if(!.)
 		return FALSE
 	to_chat(summoner, "<span class='danger'>Your [name] died somehow!</span>")
+	UnregisterSignal(summoner, COMSIG_LIVING_HEALTH_UPDATE)
 	summoner.death()
 
 
 /mob/living/simple_animal/hostile/guardian/update_health_hud()
 	if(summoner)
 		var/resulthealth
 		if(iscarbon(summoner))
-			resulthealth = round((abs(HEALTH_THRESHOLD_DEAD - summoner.health) / abs(HEALTH_THRESHOLD_DEAD - summoner.maxHealth)) * 100)
+			resulthealth = round(((summoner.health - HEALTH_THRESHOLD_CRIT) / abs(HEALTH_THRESHOLD_CRIT - summoner.maxHealth)) * 100)
```

Comment:
Does this change anything about the health value that actually gets displayed? 
If so, can you document it in the PR body and CL?

Reply:
Both of these were already in the PR body, but have now been added to CL as well